### DISCUSSION TITLE
Multiple action buttons in single table cell

### DISCUSSION THREAD

**Discussion by jwhoami** [Power-Components/livewire-powergrid#509] [Q&A]
Is it possible to have multiple row action buttons in a single table cell?

Right now using the default installation, each button is its own table cell

Thanks

**Comment by akulmehta** [comment]
I do not see any such options currently. The current rendering does feel fine though. Apart from design perspective, is there any other reason why you would want this?

**Reply by jwhoami** [reply]
> I do not see any such options currently. The current rendering does feel fine though. Apart from design perspective, is there any other reason why you would want this?

Its primarily from a design perspective. Thanks anyway

**Reply by akulmehta** [reply]
I am not sure if this will work, but you can publish the views using

```
php artisan vendor:publish --tag=livewire-powergrid-views
```

Then you can edit the `actions.blade.php` to your liking.

**Comment by luanfreitasdev** [comment]
Hello. I don't know if I understand..can you explain the usability better?. Bs5 or Tailwind?

**Reply by jwhoami** [reply]
> Hello. I don't know if I understand..can you explain the usability better?. Bs5 or Tailwind?

I am using tailwind.

Strictly speaking, I don't have an issue with how things are setup currently. However I think if there are many actions, the table grows horizontally a bit more with a new table cell for every row action. 

I know its not recommended, but I had to change the actions.blade.php to achieve what I was looking for, like so....

![image](https://user-images.githubusercontent.com/74318608/179435015-b07ac3fe-89fc-42e5-9b3e-099dab25bf1f.png)



Thanks again.
